storage/lib/src/main/java/org/calyxos/backup/storage/plugin/saf/SafStoragePlugin.kt

```diff
@@ -30,11 +30,12 @@ private const val TAG = "SafStoragePlugin"
 
 @Suppress("BlockingMethodInNonBlockingContext")
 public abstract class SafStoragePlugin(
-    private val context: Context,
+    private val appContext: Context,
 ) : StoragePlugin {
 
     private val cache = SafCache()
     protected abstract val root: DocumentFile?
+    protected abstract val context: Context
```

Comment:
Please add JavaDocs to both contexts. This is a library and it isn't obvious what those contexts are and where there are two of them.

Reply:
Done